src/web/app/src/main/java/it/geosolutions/geostore/init/GeoStoreInit.java

```diff
@@ -71,44 +71,49 @@ public void afterPropertiesSet() throws Exception {
 
         LOGGER.info("===== Starting GeoStore services =====");
         // initialize password encoding
-        initPasswordEncoding();
-        long catCnt = categoryService.getCount(null);
-        if (catCnt == 0) {
-            LOGGER.warn("No category found.");
-            if (categoryListInitFile != null) {
-                LOGGER.warn("Initializing categories from file " + categoryListInitFile);
-                initCategories(categoryListInitFile);
+        if (!Boolean.valueOf(
+                System.getProperty("it.geosolutions.geostore.init.skipInit", "false"))) {
+            initPasswordEncoding();
+            long catCnt = categoryService.getCount(null);
```

Comment:
Why do we skip the Categories initialization too here?

Reply:
Here I agree with @afabiani.